powerauth-data-adapter-client/src/main/java/io/getlime/security/powerauth/lib/dataadapter/client/DataAdapterClient.java

```diff
@@ -152,8 +153,8 @@ public ObjectResponse<InitAuthMethodResponse> initAuthMethod(String userId, Stri
      * @return Response with generated messageId.
      * @throws DataAdapterClientErrorException Thrown when client request fails or SMS could not be delivered.
      */
-    public ObjectResponse<CreateSmsAuthorizationResponse> createAndSendAuthorizationSms(String userId, String organizationId, AccountStatus accountStatus, AuthMethod authMethod, OperationContext operationContext, String lang, boolean resend) throws DataAdapterClientErrorException {
-        CreateSmsAuthorizationRequest request = new CreateSmsAuthorizationRequest(userId, organizationId, accountStatus, lang, authMethod, operationContext, resend);
+    public ObjectResponse<CreateSmsAuthorizationResponse> createAndSendAuthorizationSms(String userId, String organizationId, List<UserContact> userContacts, AccountStatus accountStatus, AuthMethod authMethod, OperationContext operationContext, String lang, boolean resend) throws DataAdapterClientErrorException {
```

Comment:
Once, in the future, it could be nice to use parameter object instead of long list of parameters.